sdcm/sct_provision/aws/cluster.py

```diff
@@ -439,4 +439,4 @@ def provision(self):
                 name=self.placement_group_name, tags=self.common_tags, region=self._region(0))
 
 
-ClusterNode.update_forward_refs()
+ClusterNode.model_rebuild()
```

Comment:
why? is that a Pydantic change?

Reply:
I would extract those into separate PR, there are too many changes here already and it is unintuitive to have this bundled with python 3.13 update